app/Models/GamesCoupdecoeur.php

```diff
@@ -34,11 +34,11 @@ class GamesCoupdecoeur extends Model
 
     protected $guarded = [];
 
-    public function game(){
+    public function game() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->